tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_search_bug.py

```diff
@@ -0,0 +1,24 @@
+from gaiatest import GaiaTestCase
+
+from gaiatest.apps.bzlite.app import BugzillaLiteStage
+
+class TestSearchBug(GaiaTestCase):
+     def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+        self.connect_to_local_area_network() # <= No detail about how to connect
+        name = 'Bugzilla Lite Stage'
+        url = 'http://bzlite-staging.herokuapp.com/manifest.webapp'
```

Comment:
This URL is also defined in `test_fillbug.py`. It's worth it to centralize it in the page class:
```py
class BugzillaLiteStage(BugzillaLite): 
     install_url = 'http://bzlite-staging.herokuapp.com/manifest.webapp'
```
and then in the test:
``` py
self.apps.install(BugzillaLiteStage.install_url) #
```